{
   Cppyy Suppression
   Memcheck:Free
   fun:free
   obj:*
   fun:_ZNK3Cpp7JitCall6InvokeEPvNS0_7ArgListES1_
   fun:_ZL11WrapperCallPvmS_S_S_
   fun:_ZN5Cppyy5CallVEPvS0_mS0_
   fun:_ZL8GILCallVPvS_PN8CPyCppyy11CallContextE
   fun:_ZN8CPyCppyy12_GLOBAL__N_112VoidExecutor7ExecuteEPvS2_PNS_11CallContextE
   fun:_ZN8CPyCppyy9CPPMethod11ExecuteFastEPvlPNS_11CallContextE
   fun:_ZN8CPyCppyy9CPPMethod7ExecuteEPvlPNS_11CallContextE
   fun:_ZN8CPyCppyy11CPPFunction4CallERPNS_11CPPInstanceEPKP7_objectmS5_PNS_11CallContextE
   fun:_ZN8CPyCppyy12_GLOBAL__N_1L13mp_vectorcallEPNS_11CPPOverloadEPKP7_objectmS4_
   fun:_PyObject_VectorcallTstate
   fun:PyObject_Vectorcall
}

# To address:
# Mismatched free() / delete / delete []
#    at 0x4878480: free (vg_replace_malloc.c:989)
#    by 0x2C82F02B: ???
#  Address 0x258d7f70 is 0 bytes inside a block of size 6 alloc'd
#    at 0x4877194: operator new[](unsigned long) (vg_replace_malloc.c:729)
#    by 0x2C82B057: ???
# 
# On Ubuntu ARM, that also causes certain tests to fail.

{
   Mismatched free()
   Memcheck:Free
   fun:free
   obj:*
}
